docs/development/API/DisplayPort.mdx

```diff
@@ -113,7 +113,7 @@ The `attribute` parameter is encoded thus.
 
 | Command        | Msg Id | Notes                                                                         |
 | -------------- | ------ | ----------------------------------------------------------------------------- |
-| MSP_DP_OPTIONS | 5      | Not used by Betaflight. Used by INAV and Ardupilot to set display resolution. |
+| MSP_DP_OPTIONS | 5      | Not used by Betaflight. Used by INAV and Ardupilot to set display resolution. 0 = SD (30x16), 1 = HD (50x18), 2 = 30x16 SD grid centered in an HD 50x18 grid (HDZero uses), 3 = 50x22 ( used by INAV / DJI WTF ) |
```

Comment:
Not sure if we need to extend this info if is not used by Betaflight.

Reply:
I do see your point...

However, BF docs are the only decent/up to date resource on this stuff; so I feel there's an argument to make them as complete as we can for the benefit of the whole community ?